synfig-studio/src/gui/workarea.h

```diff
@@ -366,6 +373,7 @@ class WorkArea : public Gtk::Grid, public Duckmatic
 	//! Returns the color of the grid
 	const synfig::Color &get_grid_color()const { return Duckmatic::get_grid_color();}
 
+	bool get_ruler_status()const { return ruler_status; }
```

Comment:
I suggest you to follow the naming style of guides: `get_show_rulers()`, `set_show_rulers()` , `toggle_rulers()` and `show_rulers`. I chose plural form because there are two rulers: horizontal and vertical.

Reply:
Sounds good